pywwt_web/annotation.py

```diff
@@ -52,7 +52,7 @@ class Circle(Annotation):
     line_color = Unicode('white', help='Assigns line color for the circle').tag(wwt='lineColor', sync=True)
     line_width = Float(1, help='Assigns line width (in pixels)').tag(wwt='linewidth', sync=True)
     radius     = Float(10, help='Sets the radius for the circle').tag(wwt='radius', sync=True)
-    sky_relative = Bool(False, help='Whether the size of the circle is absolute or relative').tag(wwt='skyRelative', sync=True)
+    sky_relative = Bool(True, help='Whether the size of the circle is absolute (in arcseconds) or relative').tag(wwt='skyRelative', sync=True)
```

Comment:
I've looked at this a little and I think we could do something more user-friendly, since we don't need to match exactly the WWT JS API. Maybe we could make it so that users can do:

```
from astropy import units as u
circle.radius = 3 * u.arcsec
circle.radius = 5 * u.pixel
```

and based on the units passed to radius, we adjust skyRelative automatically. Maybe this logic can be done inside ``_on_trait_change``. Maybe the Circle class can define its own ``_on_trait_change``, kind of like this (pseudo code, not tested)

```
def _on_trait_change(self, changed):
    if changed['name'] == 'radius':
        if changed['new'].unit.is_equivalent(u.pixel):
            self._send_msg(message here to change skyRelative to True)
            changed['new'] = changed['new'].to(u.pixel).value  # this removes the units
        else:
            self._send_msg(message here to change skyRelative to False)
            changed['new'] = changed['new'].to(u.arcsec).value
    super(Circle, self)._on_trait_changed(changed)
```

Does this make sense?

Reply:
I think so. I'll try this out and see how it goes.

Reply:
Is there a quick way to print an error message in the interpreter in JS for `wwt_json_api.js` if an invalid action is called for by the Python code? I'm trying to troubleshoot.

Reply:
@ojustino - try using ``console.log`` (use it like print in Python). For the Jupyter version it will print in the JS console.

Reply:
Is there a way to do this through qt? I'm having problems running Jupyter at all on my setup.

Reply:
I think it should work with the Qt version, it will just print the output into the terminal where you launched the Qt widget from.

Reply:
That and the other output methods I've found from searching haven't worked. The best I can manage is to print the final output sent from Python in `_send_msg()` from `qt_widget.py`, but the JavaScript fails/succeeds silently.

Reply:
@ojustino - just for info I found/fixed the issue inhttps://github.com/WorldWideTelescope/pywwt/pull/30